src/Core/Addon.php

```diff
@@ -140,7 +140,8 @@ public static function uninstall(string $addon)
 			$func();
 		}
 
-		Hook::delete(['file' => 'addon/' . $addon . '/' . $addon . '.php']);
+		// Handles both relative and absolute file paths
+		Hook::delete(["`file` LIKE '%addon/$addon/$addon.php'"]);
```

Comment:
Please use wildcards for the condition, means something like ```["`file` LIKE ?", ''%addon/$addon/$addon.php'']```

Reply:
Done.